modules/environment-roles/templates/app_base_terraform_policy.json.tpl

```diff
@@ -80,6 +80,7 @@
       "Resource": [
         "arn:aws:iam::${account_id}:policy/${app_name}_ecs_execution_policy_${environment}",
         "arn:aws:iam::${account_id}:policy/${app_name}_ecs_task_policy_${environment}",
+        "arn:aws:iam::${account_id}:policy/TDRKeycloakECSTaskPolicy${environment}",
```

Comment:
Is this an equivalent file? What's the difference between the `shared_terraform_policy_<N>.json.tpl` files and `app_base_terraform_policy.json.tpl`?

Reply:
From looking at how its used, I think its meant to be for permissions specific to ECS tasks in the environments. That said, I think the organisation of the permissions for Terraform(ing) needs to be revisited, as its gotten pretty confusing.

Reply:
I think you're right about the confusion. The reason I asked is because I can't remember how these all fit together. 🙃 

Should we add a card about simplifying it, or do you think it's worth having a discussion with with us and Sam?

Reply:
I think its worth having a discussion first, as I'm not sure what the best way is to organise the permissions.